x/gov/keeper/proposal.go

```diff
@@ -191,3 +193,33 @@ func (keeper Keeper) ActivateVotingPeriod(ctx sdk.Context, proposal types.Propos
 	keeper.RemoveFromInactiveProposalQueue(ctx, proposal.ProposalID, proposal.DepositEndTime)
 	keeper.InsertActiveProposalQueue(ctx, proposal.ProposalID, proposal.VotingEndTime)
 }
+
+func (keeper Keeper) MarshalProposal(proposal types.Proposal) []byte {
```

Comment:
Is this correct? This should return `([]byte, error)`.

Reply:
Oops!